rudder_airflow_provider/hooks/rudderstack.py

```diff
@@ -28,10 +28,14 @@ def trigger_sync(self):
         self.method = 'POST'
         sync_endpoint = f"/v2/sources/{self.source_id}/start"
         access_token = self.get_access_token()
+        headers = {
+            'authorization': f"Bearer {access_token}",
+            'Content-Type': 'text/plain'
```

Comment:
Content-Type should be `application/json`

Reply:
sure, will update it.

Reply:
updated